.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,166 @@
+    description: "What authentication flow was used when the issue appeared?"
+    multiple: true
+    options: 
+      - "Interactive (with an authentication broker)"
+      - "Interactive (with embedded or system browser)"
+      - "Integrated Windows Authentication"
+      - "Username and password (ROPC)"
+      - "Device code (browserless)"
+      - "Authorization code"
+      - "On-Behalf-Of"
+      - "Client credentials (service-to-service calls)"
+      - "Long-running On Behalf Of (OBO)"
+      - "Managed Identity"
+      - "Other"
+  validations:
+    required: true    
+
+- type: dropdown
+  attributes:
+    label: Is this a new or existing app?
+    description: "Is this a new or existing app?"
+    multiple: true
+    options: 
+      - "The app is in production, and I have upgraded to a new version of MSAL"
+      - "The app is in production, I haven't upgraded MSAL, but started seeing this issue"
+      - "This is a new app or experiment"
+  validations:
+    required: true        
+
+- type: textarea
+  attributes:
+    label: Issue description and reproduction steps
+    description: "Briefly explain the issue you are seeing. Please also provide steps to reproduce or a link to a sample which demonstrates the behavior."
+    
+- type: textarea
+  attributes:
+    label: Error message
+    description: "Please provide any error messages you are receiving along with a stack trace, if possible."
+
+- type: textarea
+  attributes:
+    label: MSAL-specific logs
+    description: "Please provide verbose level log messages from MSAL. [Learn more](https://learn.microsoft.com/azure/active-directory/develop/msal-logging-dotnet)"
+       
+- type: textarea
+  attributes:
+    label: Relevant code snippets
+    description: "Please provide relevant code snippets that can be used to reproduce the issue."
+    render: javascript
+  validations:
+    required: false
+       
+- type: textarea
+  attributes:
+    label: Expected behavior
+    description: "Please describe what you expect the behavior to be."
+  validations:
+    required: true
+    
+- type: dropdown
+  attributes:
+    label: Identity provider
+    options: 
+      - Microsoft Entra ID and/or MSA
+      - Azure B2C Basic Policy
+      - Azure B2C Custom Policy
+      - Azure Active Directory Federation Services (ADFS)
+      - Microsoft Entra External ID
+      - Other
+  validations:
+    required: true
+       
+- type: input
+  attributes:
+    label: Regression
+    description: "If this behavior worked before, please enter the last working version(s) of MSAL."
+    placeholder: "MSAL version: "
+
+- type: textarea
+  attributes:
+    label: Solution and workarounds
+    description: "Possible solution or workarounds, if you know of any."
+  validations:
+    required: false
+    
+- type: dropdown
+  attributes:
+    label: Source
+    description: "Are you an internal or external customer?"
+    options: 
+      - External (outside Microsoft)
+      - Internal (Microsoft)
```

Comment:
Hmm, I think I'd remove this. It may make it seem like we're prioritizing internal? Plus, we'd still have to go through steps to get peoples email to contact them and such.